<ui version="4.0" >
 <author>Petr Vanek &lt;petr@scribus.info></author>
 <comment>General prefs panel</comment>
 <class>TabGeneral</class>
 <widget class="QWidget" name="TabGeneral" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TabGeneralBase</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="Q3GroupBox" name="groupBox1" >
     <property name="title" >
      <string>User Interface</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="1" >
           <widget class="QComboBox" name="guiLangCombo" />
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="textLabel1" >
            <property name="text" >
             <string>&amp;Language:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>guiLangCombo</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QComboBox" name="GUICombo" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="textLabel2" >
            <property name="text" >
             <string>&amp;Theme:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>GUICombo</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="2" column="1" >
           <widget class="QSpinBox" name="spinTimeout" >
            <property name="suffix" >
             <string> ms</string>
            </property>
            <property name="maximum" >
             <number>1500</number>
            </property>
            <property name="minimum" >
             <number>10</number>
            </property>
            <property name="singleStep" >
             <number>10</number>
            </property>
            <property name="value" >
             <number>150</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QSpinBox" name="GFsize" >
            <property name="suffix" >
             <string> pt</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="textLabel3" >
            <property name="text" >
             <string>Time before a Move or Resize starts:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>showSplashCheckBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="textLabel4_2" >
            <property name="text" >
             <string>Font Size (&amp;Palettes):</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>GTFsize</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="textLabel4" >
            <property name="text" >
             <string>&amp;Font Size (Menus):</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>GFsize</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QSpinBox" name="GTFsize" >
            <property name="suffix" >
             <string> pt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="textLabel5" >
            <property name="text" >
             <string>&amp;Wheel Jump:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>wheelJumpSpin</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QSpinBox" name="wheelJumpSpin" />
          </item>
          <item row="1" column="1" >
           <widget class="QSpinBox" name="recentDocs" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="textLabel6" >
            <property name="text" >
             <string>&amp;Recent Documents:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>recentDocs</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="startUpDialog" >
          <property name="text" >
           <string>Show Startup Dialog</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showSplashCheckBox" >
          <property name="text" >
           <string>Show Splashscreen on Startup</string>
          </property>
          <property name="shortcut" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="groupBox2" >
     <property name="title" >
      <string>Paths</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="2" >
       <widget class="QPushButton" name="FileC3" >
        <property name="text" >
         <string>Ch&amp;ange...</string>
        </property>
        <property name="shortcut" >
         <string>Alt+A</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="textLabel10" >
        <property name="text" >
         <string>Document &amp;Templates:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>DocumentTemplateDir</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel7" >
        <property name="text" >
         <string>&amp;Documents:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>Docs</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel8" >
        <property name="text" >
         <string>&amp;ICC Profiles:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>ProPfad</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QPushButton" name="FileC4" >
        <property name="text" >
         <string>Cha&amp;nge...</string>
        </property>
        <property name="shortcut" >
         <string>Alt+N</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="DocumentTemplateDir" >
        <property name="minimumSize" >
         <size>
          <width>268</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="textLabel9" >
        <property name="text" >
         <string>&amp;Scripts:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>ScriptPfad</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="FileC2" >
        <property name="text" >
         <string>C&amp;hange...</string>
        </property>
        <property name="shortcut" >
         <string>Alt+H</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="FileC" >
        <property name="minimumSize" >
         <size>
          <width>88</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>&amp;Change...</string>
        </property>
        <property name="shortcut" >
         <string>Alt+C</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="ProPfad" >
        <property name="minimumSize" >
         <size>
          <width>268</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="Docs" >
        <property name="minimumSize" >
         <size>
          <width>268</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="ScriptPfad" >
        <property name="minimumSize" >
         <size>
          <width>268</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
